tvtk/__init__.py

```diff
@@ -6,6 +6,10 @@
 """
 
 from os.path import exists, join, dirname, isdir
+import logging
+
+# use a null handler when used as a library
+logging.getLogger('tvtk').addHandler(logging.NullHandler())
```

Comment:
Adding a null handler is suggested for libraries that use the python logging framework so that there is always a Handler  (see https://docs.python.org/2/howto/logging.html#configuring-logging-for-a-library).